scripts/exregional_run_analysis.sh

```diff
@@ -631,6 +639,28 @@ HYBENSINFO=${FIX_GSI}/${HYBENSINFO_FN}
 OBERROR=${FIX_GSI}/${OBERROR_FN}
 BERROR=${FIX_GSI}/${BERROR_FN}
 
+
+if [[ ${gsi_type} == "ANALYSIS" && ${ob_type} == "AERO" ]]; then
+  ANAVINFO=${FIX_GSI}/${ANAVINFO_SD_FN}
+  CONVINFO=${FIX_GSI}/${CONVINFO_SD_FN}
+  BERROR=${FIX_GSI}/${BERROR_SD_FN}
+  miter=1
+  niter1=100
+  niter2=0
+  ifhyb=.false.
+  ifsd_da=.true.
+  l_hyb_ens=.false.
+  nummem=0
+  beta1_inv=0.0
+  i_use_2mQ4b=0
+  i_use_2mT4b=0
```

Comment:
655 and 656 change to:
i_use_2mQ4**B**=0
i_use_2mT4**B**=0

Reply:
@guoqing-noaa Fixed. Thanks.